bobtimus/src/lib.rs

```diff
@@ -270,32 +275,92 @@ where
     /// We return the range of possible loan terms to the borrower.
     /// The borrower can then request a loan using parameters that are within our terms.
     pub async fn handle_loan_offer_request(&mut self) -> Result<LoanOffer> {
-        Ok(LoanOffer {
+        Ok(self.current_loan_offer())
+    }
+
+    fn current_loan_offer(&mut self) -> LoanOffer {
+        LoanOffer {
             rate: self.rate_service.latest_rate(),
             // TODO: Dynamic fee estimation
             fee_sats_per_vbyte: Amount::from_sat(1),
-            // TODO: Send sats over the wire and refactor waves reducer to use amount classes
             min_principal: LiquidUsdt::from_str_in_dollar("100")
                 .expect("static value to be convertible"),
             max_principal: LiquidUsdt::from_str_in_dollar("10000")
                 .expect("static value to be convertible"),
+            // TODO: Maximum LTV to be decided by a model
             max_ltv: dec!(0.8),
-            // TODO: Dynamic interest based on current market values
-            interest: vec![Interest {
-                term: 30,
-                interest_rate: dec!(0.15),
-                collateralization: dec!(1.5),
-            }],
-        })
+            // TODO: Interest to be decided by a model
+            base_interest_rate: dec!(0.05),
+            // TODO: Potentially fine-tune the model with these values
+            terms: vec![
+                Term {
+                    days: 30,
+                    interest_mod: Decimal::ZERO,
+                },
+                Term {
+                    days: 60,
+                    interest_mod: Decimal::ZERO,
+                },
+                Term {
+                    days: 120,
+                    interest_mod: Decimal::ZERO,
+                },
+            ],
+            collateralizations: vec![
+                Collateralization {
+                    collateralization: dec!(1.5),
+                    interest_mod: Decimal::ZERO,
+                },
+                Collateralization {
+                    collateralization: dec!(2.0),
+                    interest_mod: Decimal::ZERO,
+                },
+            ],
+        }
     }
 
     /// Handle the borrower's loan request in which she puts up L-BTC as
     /// collateral and we lend L-USDt to her which she will have to
     /// repay in the future.
```

Comment:
Similarly to commit messages, idiomatic rustdoc has a subject line with more fine-grained documentation in another paragraph: https://doc.rust-lang.org/rustdoc/how-to-write-documentation.html#documenting-components